components/fieldgroups/location/location-sports-entertainment-event-venue.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+  "meta:license": [
+    "Copyright 2022 Bounteous. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixin/sports-entertainment-event-venue",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Sports Entertainment Event Venue",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/location"],
+  "meta:tags": {
+    "industry": ["travel_and_hospitality"]
+  },
+  "description": "For Sports and Entertaiment Event Venue information, such as a concert hall or sports arena. ",
+  "definitions": {
+    "sports-entertainment-event-venue": {
+      "properties": {
+        "xdm:eventVenue": {
+          "title": "Event Venue",
+          "type": "object",
+          "description": "Details of a live event venue physical location.",
+          "properties": {
+            "xdm:DMAs": {
+              "description": "The list of associated DMAs (Designated Market Areas) of the venue",
+              "items": {
+                "meta:xdmType": "string",
+                "type": "string"
+              },
+              "meta:xdmType": "array",
+              "title": "DMA List",
+              "type": "array"
+            },
+            "xdm:currency": {
+              "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+              "description": "",
+              "meta:xdmType": "object",
+              "title": "Currency",
+              "type": "object"
+            },
+            "xdm:markets": {
```

Comment:
every Property needs to have a title. Please see other properties too where we have a missing title